packages/doc/content/components/components/navigationmenu/props.ts

```diff
@@ -225,6 +224,13 @@ export const BottomItemsProps = [
     type: { name: 'node' },
     required: true,
   },
+  {
+    name: 'fill',
```

Comment:
As this is 'Color used as background' i think it would be better called `backgroundColor`.
```suggestion
    name: 'backgroundColor',
```

Reply:
I am not sure what would be the best approach - WDYT @matheushdsbr? 

Just a note: we already use a `fill` prop in `NavigationMenu.Switcher`

Reply:
After talk with @matheus-pereira we decided that doesn't make sense live this background color as a component prop. Therefor it was removed from component.